Docs: Add warning on UUIDs not being available before 10.2.0 (#91372)
grafana/grafana: docs/sources/administration/roles-and-permissions/access-control/rbac-fixed-basic-role-definitions/index.md

The following table has the existing built-in fixed role definitions. Other fixed roles might be added by plugins installed in Grafana.
The UUID presented here can be used as an identifier for [Terraform provisioning](../rbac-terraform-provisioning).

{{% admonition type="caution" %}}
These UUIDs might not be available if your instance was populated before Grafana v10.2.0. You can use the roles API to determine the UUIDs for the roles you need. You can learn more about this [here]({{< relref "../manage-rbac-roles" >}}).

> {{% /admonition %}}

Author: Thanks for the review @jdbaldry , it's definitely less ambiguous. I think we can keep it as you rephrased it

> {{% /admonition %}}

Reviewer: Thanks for taking a look. So is it correct to say it's definitely not available for instances created before v10.2.0?

If so, then you will want something like:

These UUIDs won't be available if your instance was created before Grafana v10.2.0.

The following table has the existing built-in fixed role definitions. Other fixed roles might be added by plugins installed in Grafana.
The UUID presented here can be used as an identifier for [Terraform provisioning](../rbac-terraform-provisioning).

{{< admonition type="caution" >}}

> These UUIDs might not be available if your instance was created before Grafana v10.2.0.

Reviewer 2: ```suggestion
These UIDs might not be available if your instance was created before Grafana v10.2.0.
```
the ids are not UUIDs... just unique withint org

Reviewer 3: I think they are unique across all orgs as the role itself is unique across all orgs.

> These UUIDs might not be available if your instance was created before Grafana v10.2.0.

Author: ```suggestion
These UUIDs won't be available if your instance was created before Grafana v10.2.0.
```
